### PR TITLE
Add recipe for bioconductor-pharmacogx

### PR DESCRIPTION
**Pull request by npavlovikj**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bioconductor-pharmacogx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  commands:
+    - '$R -e "library(''{{ name }}'')"'
+about:
+  home: 'http://bioconductor.org/packages/{{ bioc }}/bioc/html/{{ name }}.html'
+  license: Artistic-2.0
+  summary: 'Contains a set of functions to perform large-scale analysis of pharmacogenomic data.'
+
+extra:
+  skip-lints:
+    - should_not_use_fn
```
Comment (corburn) [inline recipes/bioconductor-pharmacogx/meta.yaml]: `fn` is no longer required with conda3

Comment (npavlovikj) [reply recipes/bioconductor-pharmacogx/meta.yaml]: Thanks for the catch @corburn - the recipe is fixed now.

**Comment by corburn** [comment]
I'd merge the recipe (looks good), but I don't yet know the significance of the bioc version as it pertains to conda.

**Comment by dpryan79** [comment]
This has since been added.

**Pull request closed without merging**